### ISSUE TITLE
add more results plots to multi-scenario NB (things not done in #64)

### ISSUE THREAD

**Issue by gplssm** [windnode/WindNODE_ABW#180]
Notes
- In the current version there're many plots showing all scenarios which makes it somehow hard to grasp. For some plots we should pick out only relevant scenarios.
- Implement according to priority (1=high/must-have, 2=medium, 3=low)
- Unless otherwise specified, spatial resolution is ABW

### 1.1 Installed Capacities Electricity/Heat

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 1.1.3 | Installed Capacity Wind+PV (PV=ground+roof) | - | dotplot | x: wind, y: PV, maybe good to quickly grasp the major tech | 3 |  |

1.1.1: If el+th is too much in one plot, split into 2 plots.

### 2.1 Available areas: Scenarios

Overview / cumulative sums:

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 2.1.0.1 | Potential Areas | GW, % (2 axes) | barplot grouped | similar to plot `Relative Required Area` in single scenario jupy nb but with abs. numbers -> abs. in GW, rel. in % related to legal SQ areas | 1 | x |

More detailed analysis in 2.1.1-2.1.3 .

#### 2.1.1 Available areas: Wind Energy

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 2.1.1.1 | Potential Areas Wind | - | geoplot | All avail. areas (geometries) plotted on mun shapes (s1000f0, s1000f1, s500f0, s500f1), geoms are avail. in DF `region.pot_areas_wec`, please use 'sq' for s1000f0 | 2 |  |
| 2.1.1.2 | Potential Areas Wind per mun | ha | barplot grouped: (s1000f0, s1000f1, s500f0, s500f1) per scn in on group | x: mun , geoms are avail. in DF `region.pot_areas_wec`, please use 'sq' for s1000f0 | 1 |  |

#### 2.1.2 Available areas: Ground-Mounted PV

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 2.1.2.1 | Potential Areas PV ground: railways | ha | geoplot | All avail. areas (geometries) plotted on mun shapes side by side (H, HS), geoms are avail. in DF `region.pot_areas_pv` | 2 |  |
| 2.1.2.2 | Potential Areas PV ground: motorways | ha | geoplot | All avail. areas (geometries) plotted on mun shapes side by side (H, HS), geoms are avail. in DF `region.pot_areas_pv` | 2 |  |
| 2.1.2.3 | Potential Areas PV ground: agricultural | ha | geoplot | All avail. areas (geometries) plotted on mun shapes side by side (H, HS), geoms are avail. in DF `region.pot_areas_pv` | 2 |  |
| 2.1.2.4 | Potential Areas PV ground: area sums | ha |  barplot grouped (railway, motorway, agri per scn in on group) | x: scenarios | 1 |  |

#### 2.1.3 Available areas: Roof-Mounted PV

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 2.1.3.1 | Potential Areas PV roof | ha | geoplot | Area sum Summe Flächen pro Gemeinde | 1 |  |

### 2.2 Area usage

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 2.2.1 | Rel. Area Usage Wind (cum. for ABW) | % | barplot grouped | For each scenario: rel. usage in relation to land use scenario of scenario, sorted by the 4 land use scenarios (see photo below) | 1 |  |
| 2.2.2 | Rel. Area Usage Wind (cum. for ABW) | % | barplot | Rel. usage in all scenarios in relation to the total region's area ([362900 ha](https://www.planungsregion-abw.de/index.php/raumbeobachtung/daten-und-fakten/)) | 1 |  |
| 2.2.3 | Rel. Area Usage PV ground (cum. for ABW) | % | barplot grouped | For each scenario: rel. usage in relation to land use scenario of scenario, sorted by the 6 land use scenarios (see photo below) | 1 |  |
| 2.2.4 | Rel. Area Usage PV ground (cum. for ABW) | % | barplot | Rel. usage in all scenarios in relation to the total region's area ([362900 ha](https://www.planungsregion-abw.de/index.php/raumbeobachtung/daten-und-fakten/)) | 1 |  |
| 2.2.5 | Rel. Area Usage PV roof (cum. for ABW) | % | barplot | Rel. usage in all scenarios | 1 |  |

Example Wind 2.2.1+2.2.2 (PV ground 2.2.3+2.2.4 analogously):
![plots1](https://user-images.githubusercontent.com/15910173/89951937-9a159980-dc2c-11ea-826c-31bdb11a2c14.jpg)

### 3.1 Energy Exchange

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 3.1.1 | Import and Export (National Grid) | GWh | barplot grouped | x: scenarios, **could be merged into 3.2.1?** | 1 |  |
| 3.1.2 | Intra-Regional Import and Export | GWh | barplot grouped | x: scenarios | 1 |  |

### 3.2 Autarky

(#85 needs to be done first)

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 3.2.1 | Demand+supply | GWh, % | barplot grouped | like plot `Electrical Autarky per Municipality (Energy Balance) and total region` in single scenario jupy nb but with scenarios instead of muns  | 1 |  |
| 3.2.2 | Autarky: energy + time | % | barplot grouped | time: 2 options: 1) mean of all time step autarky values, 2) perc. of autark hours -> to be specified after #85 | 1 |  |
| 3.2.3 | Autarky: energy violin | % | violin plot | like plot `Relative Electrical Autarky per Municipality (Energy Balance)` in single scenario jupy nb but with scenarios instead of muns | 2 |  |

## 4 Energy Mix

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 4.3 | Full-Load Hours Electricity | h | barplot grouped by tech (wind, pv ground, pv roof, bio, ...)  | like plot `Power Generation and Demand` in single scenario jupy nb but with scenarios instead of muns | 2 |  |

## 5 Emissions

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 5.2 | Emissions relative | % | barplot stacked by sector (el, th, stor, grid), scaled to 100% | x: scenarios | 3 |  |

## 6 Costs

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 6.1 | LCOE and LCOH | €/MWh | barplot grouped per scn with el+th | x: scenarios | 1 |  |

## 7 Power Grid

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 7.1 | Line Loading | % | whisker box plot, like `DSM Activation` plot in single scenario jupy nb  | x: scenarios | 1 |  |

## 8 Flexibility

It'd be good to have an overview plot on the flex. to quickly see the configuration/character of each scenario. I propose either 8.0.1 or 8.0.2. Alternatively, we could move this to the scenario overview (maybe better).

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 8.0.1 | Flex. Character | €/MWh | PairGrid dot plot (as in 0) showing the flex. character of scenario (RES power, DSM, autarky, pth stor, batteries, eff. el., eff. th.) | x: scenarios | 2 |  |
| 8.0.2 | Flex. Character | €/MWh | [radar chart](https://plotly.com/python/radar-chart/) showing the flex. character of scenario (RES power, DSM, autarky, pth stor, batteries, eff. el., eff. th.), N charts we assign the scenarios to, e.g. BAThigh+BATlimit in one radar and so on | x: scenarios | 2 |  |

### 8.1 Thermal Storages

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 8.1.1 | Storage use | - | whisker box plot grouped by scn with dec+cen, like `DSM Activation` plot in single scenario jupy nb  | x: scenarios | 1 |  |

### 8.2 Batteries

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 8.2.1 | Storage use | - | whisker box plot grouped by scn with large+small, like `DSM Activation` plot in single scenario jupy nb  | x: scenarios | 1 |  |

### 8.3 DSM

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 8.3.1 | Total Sum (Ratio DSM / demand hh) | % | bar plot | x: scenarios | 1 |  |

## 9 Energy Exchange

As proposed in https://github.com/windnode/WindNODE_ABW/issues/96#issuecomment-670477150 I integrated this section into 3.

**Comment by nesnoj** [comment]
Only implement a selection of plots listed above

**Comment by nesnoj** [comment]
Ok, I went through the first part of the plots and from my point of view these ones should be implemented:
(The second part will follow later today)

## 2 Area required by RES

### 2.1 Available areas: Scenarios

#### 2.1.1 Available areas: Wind Energy

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 2.1.1.1 | Potential Areas Wind | - | 4 geoplots (2x2) | All avail. areas (geometries) plotted on mun shapes (s1000f0, s1000f1, s500f0, s500f1), geoms are avail. in DF `region.pot_areas_wec`, please use 'sq' for s1000f0 | 1 |  |

In 2.1.1.1 the absolute available area per mun should either be displayed by coloring the muns (underneath the area geoms) or as numbers.

#### 2.1.2 Available areas: Ground-Mounted PV

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 2.1.2.1 | Potential Areas PV ground | - | geoplot (2x1) | All avail. areas (geometries) plotted on mun shapes (H, HS), types (motorways, railways, agricultural), geoms are avail. in DF `region.pot_areas_pv` | 1 |  |

- 2.1.2.1 like [this plot](https://stemp-abw.readthedocs.io/en/dev/areas_and_potentials.html#id3) in the StEmp ABW docs. If possible add available area per mun as described above (2.1.1.1).
- Add note that the shown agri areas are 100%, we use 0.1% and 1% in our scenarios.
- A comprehensive description of how those areas are determined see [StEmp ABW docs](https://stemp-abw.readthedocs.io/en/dev/areas_and_potentials.html#freiflachen-photovoltaikanlagen).

#### 2.1.3 Available areas: Roof-Mounted PV

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 2.1.3.1 | Potential Areas PV roof | ha | geoplot | Area sum Summe Flächen pro Gemeinde | 1 |  |

## 3 Energy Exchange and Autarky

### 3.1 Energy Exchange

|# | Größe | Einheit | Plot/Tabelle | Kommentar | Prio | Done? |
|-------|----------|-----------|-----------------|-----------------|---|---|
| 3.1.1 | Import and Export (National Grid) | GWh | barplot grouped | x: scenarios, **could be merged into 3.2.1?** | 1 |  |
| 3.1.2 | Intra-Regional Import and Export | GWh | barplot grouped | x: scenarios | 1 |  |

**Comment by nesnoj** [comment]
Ok @nailend @gplssm  I'm done with the plots and pushed some amendments to dev.
Also I added some points to be discussed in the new list above and to the template.
